.gitignore

```diff
@@ -1,10 +1,3 @@
 .bin/
-build/
-build-nodejs-16/
-build-nodejs-18/
-build-nodejs-20/
-build-java-8/
-build-java-11/
-build-java-17/
-build-java-21/
-stack/images.json
+builds/
+stacks/stack/images.json
```

Comment:
I'm thinking that since we now have the stacks directory it would makes sense for images.json to be in stacks/images.json as opposed to nested into one of the directories is specific to a stack? Maybe I'm forgetting some reason that won't work?

Reply:
Makes sense, moved it and tested it. Works. Thank you